plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -460,6 +469,26 @@ public OptionalLong delete(ConnectorSession session, JdbcTableHandle handle)
         return deletedRowsCount;
     }
 
+    @Managed
+    public void flushCache()
+    {
+        schemaNamesCache.invalidateAll();
+        tableNamesCache.invalidateAll();
+        tableHandleCache.invalidateAll();
+        columnsCache.invalidateAll();
+        statisticsCache.invalidateAll();
+    }
+
+    private void validateCacheConfiguration(Duration metadataCachingTtl, long cacheMaximumSize)
```

Comment:
Does this get executed during server startup? Or only once the caching client gets created (which can be later)? If not then maybe add a `@PostConstruct public void validate()` to BaseJdbcConfig itself.